idea-codestyle.xml

```diff
@@ -0,0 +1,5 @@
+<code_scheme name="Default" version="173">
```

Comment:
Why do we need this file?

Reply:
It was meant for IntelliJ IDEA, but for now we won't be using it. Removed.